### DISCUSSION TITLE
Questions about the DLC

### DISCUSSION THREAD

**Discussion by hadiels** [Russifiers-for-Humans/Crusader-Kings-3-DLC#3] [General]
Hello, I just wanted to know if you're uploading all the DLCs for the CK3 game, is it cracked or do I have to buy the base game in order to play with the DLCs? while connecting to the paradox launcher and playing online including the DLCs?

Thank you for your contribution.
Regards.

**Comment by 55TeMeR55** [comment]
I download all add-ons for SC3, You need the base game to play with DLC. Without a launcher or with a launcher to run it is up to you, it works both ways.
